urbanopt/geojson-modelica-translator
# Update within clause on project export

#### Any background context you want to provide?
Dymola doesn't show the within clause of the mo file in the text view. If the file is loaded in dymola as part of a project, then the within clause is automatically added/updated. 

#### What does this PR accomplish?
* force the existence of the within clause to be part of the project generation
* Update the copy_required_mo_files method to allow for the within clause to be added
* Remove the PartialBuilding.mo file as it appears to have reappeared in the library even though we are using the one out of MBL now.

#### How should this be manually tested?
* py.test should suffice
* upon completion of py.test running, open a mo file (e.g., tests/model_connects/output/district_1/Spawn...mo and verify that the within clause at the top of the file says `within districts_1.districts`.

#### What are the relevant tickets?
#156 

#### Screenshots (if appropriate)
